lib/Backend/Opnd.cpp

```diff
@@ -3180,11 +3207,14 @@ Opnd::Dump(IRDumpFlags flags, Func *func)
         IndirOpnd *indirOpnd = this->AsIndirOpnd();
 
         Output::Print(_u("["));
-        indirOpnd->GetBaseOpnd()->Dump(flags, func);
+        if (indirOpnd->GetBaseOpnd())
+        {
+            indirOpnd->GetBaseOpnd()->Dump(flags, func);
+            Output::Print(_u("+"));
```

Comment:
Same comment as above about result of GetBaseOpnd -- save to variable before the if